packages/shortcuts-extension/schema/plugin.json

```diff
@@ -428,6 +428,17 @@
       },
       "type": "object"
     },
+    "filemenu:close-and-cleanup": {
+      "default": {},
+      "properties": {
+        "command": { "default": "filemenu:close-and-cleanup" },
+        "keys": { "default": ["Ctrl Shift Q"] },
+        "selector": { "default": "body" },
```

Comment:
Thanks @melsyt! I wonder if we can be a bit narrower with the selector here. `docmanager:close` uses `jp-Activity` so that it only applies when you are focused on an activity in the main area (as opposed do something in the side panel or menu bar).

Reply:
Updated selector. Hope I understood you correctly!